src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/integration/TestWebClientConfig.kt

```diff
@@ -0,0 +1,39 @@
+package uk.gov.justice.digital.hmpps.managerecallsapi.integration
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.boot.test.context.TestConfiguration
+import org.springframework.context.annotation.Bean
+import org.springframework.http.MediaType.APPLICATION_JSON
+import org.springframework.http.codec.json.Jackson2JsonDecoder
+import org.springframework.http.codec.json.Jackson2JsonEncoder
+import org.springframework.web.reactive.function.client.WebClient
+import uk.gov.justice.digital.hmpps.managerecallsapi.config.ManageRecallsApiJackson
+
+// Create WebClients without MetricsCustomizer
```

Comment:
As above, I haven't understood why we need this - and for these two end-points only.

Reply:
These seem to be the only 2 that have Integration tests.

Reply:
Ah, yes, because we use the real end-points for tests of police-forces and bank-holidays.